.travis.yml

```diff
@@ -24,12 +23,17 @@ install:
 before_script:
     - export PATH=$HOME/gopath/bin:$PATH
 
-script:
-    - make .gitvalidation
-    - make verify
-    - make binaries
-    - make test
-    - make test-cri
+jobs:
+  include:
+    - stage: Static Check
+      script: 
+        - make .gitvalidation
+        - make verify
+    - stage: Build and Test
+      script:
+        - make binaries
+        - make test
+        - make test-cri
```

Comment:
yes make test-cri has a dependency on the binaries. make test should not .. if that matters.